packages/core/src/internal/clients/auth/AbstractAuthClient.ts

```diff
@@ -22,36 +22,55 @@ export abstract class AbstractAuthClient implements IAuthClient {
   }
 
   /**
-   * Generates a new auth token, along with a refresh token to refresh the auth token before expiry.
+   * Generates a new API key, along with a refresh token to refresh the API key before expiry.
    *
-   * @param {TokenScope} scope - controls the permissions that the new token will have
+   * @param {TokenScope} scope - controls the permissions that the new key will have
    * @param {string} expiresIn - How long the token is valid for in epoch timestamp.
```

Comment:
```suggestion
   * @param {string} expiresIn - How long the api key should be valid for in epoch timestamp.
```

Reply:
fixed in https://github.com/momentohq/client-sdk-javascript/pull/838/commits/f85c3a7deb1fdb69f483f74caf7e5bc9f852dd36 , thanks!